### PR TITLE
Bugfix/login without tenant

### PR DESCRIPTION
**Pull request by Aringoaway**
I tested it:
- without AuthModule
- without SeaCatAuthModule a + tenant [ ] and tenant [firsttenant, secondtenant]
- without devConfig
- without tenant in devConfig

### CODE REVIEW

**Comment by Aringoaway** [comment]
If there is no user without a tenant, they will be shown a card with an 'Back to login' button.
https://user-images.githubusercontent.com/65866093/194518485-3e8f2a14-a262-48d2-aeea-e39963338248.mp4

File: src/modules/tenant/service.js [hunk 1]
```diff
@@ -40,6 +40,11 @@ export default class TenantService extends Service {
 
 		// In case if the tenant list from userinfo is undefined or empty remove tenant parameter from URL
 		if ((!tenants_list) || (tenants_list.length == 0)) {
+			this.App.Store.dispatch({
+				type: types.TENANTS_CHANGED,
+				undefined,
+				current: null
+			});
```
Comment (ateska) [inline src/modules/tenant/service.js:47]: Is this needed?

**Review by ateska — CHANGES_REQUESTED** [review]
Small remark.

File: src/modules/tenant/service.js (hunk 1 above)
Comment (Aringoaway) [reply src/modules/tenant/service.js:47]: @ateska Yes, is an important part of this change

Comment (Aringoaway) [reply src/modules/tenant/service.js:47]: @ateska we need it at a time when no tenants are assigned. And we write it as tenant - undefined to distinguish situations when to show this Logout card

Comment (ateska) [reply src/modules/tenant/service.js:47]: I still don't get it.
Let's discuss this latest on review, pls.

**Review by ateska — CHANGES_REQUESTED** [review]

**Comment by Pe5h4** [comment]
This branch is staled

**Comment by ateska** [comment]
Yeah. Close it if it is not needed to be merged.

I still don't get why we need to set a tenant and then refresh the application ... that was my only serious question here.

**Comment by Pe5h4** [comment]
@Aringoaway ^^^

**Pull request closed without merging**